libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/FreeDriveMapBinder.kt

```diff
@@ -26,7 +27,8 @@ internal class FreeDriveMapBinder(
             DropInNavigationCamera(
                 navigationViewContext.viewModel.cameraState,
                 mapView
-            )
+            ),
+            RouteFetchComponent(mapView.context),
```

Comment:
Now that we have the `RouteFetchComponent` we can remove the `RouteLineComponent` from FDBinder.

Reply:
removed